Automattic/wp-calypso
# Fix duplicate @wordpress/data package. Also update all floating deps

#### Changes proposed in this Pull Request

* Fix duplicate @wordpress/data package.
* Also update all floating deps

#### Testing instructions

* `useSelect` should start working reliably again

See https://github.com/Automattic/wp-calypso/pull/38880#issuecomment-577687276
